Classes/Service/ConfigurationService.php

```diff
@@ -64,10 +68,10 @@ protected static function redirectPageIdExists($configuration): bool
      * @return int
      * @throws Exception
      */
-    public static function getRootPageId(): int
+    public function getRootPageId(): int
     {
-        $queryBuilder = GeneralUtility::makeInstance(ConnectionPool::class)
-                                      ->getQueryBuilderForTable('pages');
+        $queryBuilder = (GeneralUtility::makeInstance(ConnectionPool::class))
```

Comment:
braces can be removed like it was before

Reply:
But then my IDE is "complaining" and until PHP 8.4 is the minimum, this doesn't hurt.

Reply:
if you want, you can even inject the ConnectionPool via DI.